Dockerfile

```diff
@@ -5,7 +5,9 @@ COPY . .
 RUN mkdir -p .data/results
 RUN apt-get update
 RUN apt-get install -y < requirements-apt-get.txt
-RUN pip3 install --upgrade pip
+RUN pip3 install --upgrade pip 
+# Below command is only used here separately as it will show error if placed in requirements.txt 
+RUN pip3 install mysql-connector-python  
```

Comment:
this must be in the requirements.txt

Reply:
I tried this method earlier but it didn't worked, that's why I found this solution and ran `RUN pip3 install mysql-connector-python` , as a separate line in dockerfile.